src/lang/english.txt

```diff
@@ -1021,6 +1010,25 @@ STR_GAME_OPTIONS_FONT_ZOOM_DROPDOWN_NORMAL                      :Normal
 STR_GAME_OPTIONS_FONT_ZOOM_DROPDOWN_2X_ZOOM                     :Double size
 STR_GAME_OPTIONS_FONT_ZOOM_DROPDOWN_4X_ZOOM                     :Quad size
 
+STR_GAME_OPTIONS_GRAPHICS                                       :{BLACK}Graphics
+
+STR_GAME_OPTIONS_FULLSCREEN                                     :{BLACK}Fullscreen
+STR_GAME_OPTIONS_FULLSCREEN_TOOLTIP                             :{BLACK}Check this box to play OpenTTD fullscreen mode
+
+STR_GAME_OPTIONS_RESOLUTION                                     :{BLACK}Screen resolution
+STR_GAME_OPTIONS_RESOLUTION_TOOLTIP                             :{BLACK}Select the screen resolution to use
+STR_GAME_OPTIONS_RESOLUTION_OTHER                               :other
+
+STR_GAME_OPTIONS_VIDEO_ACCELERATION                             :{BLACK}Hardware acceleration
+STR_GAME_OPTIONS_VIDEO_ACCELERATION_TOOLTIP                     :{BLACK}Check this box to allow OpenTTD to try to use hardware acceleration. A changed setting will only be applied upon game restart
+STR_GAME_OPTIONS_VIDEO_ACCELERATION_RESTART                     :{WHITE}The setting will only take effect after a game restart
```

Comment:
It'd be better if these strings weren't moved - then eints won't needlessly invalidate 7 strings